openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-dataextractor/src/tools/DataExtractor/DataExtractor.vue

```diff
@@ -426,7 +426,11 @@ export default {
       this.saveDefaultConfig(this.currentConfig)
     },
     cmdOrTlm: function () {
-      this.saveDefaultConfig(this.currentConfig)
+      if (this.items.length === 0) { // just to prevent the save from happening twice
+        this.saveDefaultConfig(this.currentConfig)
+      } else {
+        this.items = []
+      }
```

Comment:
I'm not sure why you're doing this if check. Don't we always want to clear the items and then save the config?

Reply:
Clearing the items does save the config because of the `items` watcher. This check makes sure that we only save the config once instead of spamming that function